AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -511,9 +514,21 @@ - (ASCellNode *)dataController:(ASDataController *)dataController nodeAtIndexPat
   return node;
 }
 
-- (CGSize)dataController:(ASDataController *)dataController constrainedSizeForNodeAtIndexPath:(NSIndexPath *)indexPath
+- (ASSizeRange)dataController:(ASDataController *)dataController constrainedSizeForNodeAtIndexPath:(NSIndexPath *)indexPath
 {
-  CGSize restrainedSize = self.bounds.size;
+  ASSizeRange restrainedSize;
```

Comment:
I was just trying to keep the old name and thus minimize the diff. But constrainedSize is better indeed.